### DISCUSSION TITLE
No in Github notifications

### DISCUSSION THREAD

**Discussion by Mimicthis** [community/community#53400] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello I recieve in Github notifications, I only get them on email, why?

**Comment by airtower-luna** [comment]
Check your notification settings (https://github.com/settings/notifications). You can select there how you'll get different kinds of notifications.

**Reply by Mimicthis** [reply]
Thank you. How to find where I've posted this? I have posted this and one question and I couldn't find it until you replied.

**Comment by NJacksonDev** [comment]
Here is an article which explains how you can configure notifications.

https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications
